### ISSUE TITLE
Include reviewers' comments and authors' responses into the ICLR dataset

### ISSUE THREAD

**Issue by pariskang** [berenslab/iclr-dataset#1]
Could you please provide the reviewer comments and author responses from the ICLR submission? This would greatly help in understanding the whole process of how the authors addressed it. Thank you in advance for your assistance.

**Comment by dkobak** [comment]
???

What are you talking about? This paper has not been submitted to ICLR.

**Comment by pariskang** [comment]
It may be beneficial to include comments and responses in the dataset description to illustrate the complete process of ICLR papers being accepted or rejected.

**Comment by dkobak** [comment]
Oh, do you mean that you would like to have reviewers' comments and authors' responses for each ICLR submission provided as part of our dataset?

**Comment by pariskang** [comment]
Yep, I think it's important for adding these content to make a more compherensive dataset.
